layouts/partials/head.html

```diff
@@ -16,8 +16,12 @@
 
 <!-- Icons -->
 <link rel="apple-touch-icon-precomposed" sizes="144x144" href="{{ "img/apple-touch-icon-144-precomposed.png" | absURL }}">
+{{ if isset .Site.Params "favicon" }}
+<link rel="shortcut icon" href="{{ .Site.BaseURL }}{{ .Site.Params.favicon }}">
+{{ else }}
 {{ with $.Site.Params.ThemeColor }}
```

Comment:
```suggestion
{{ else }}
  <link rel="shortcut icon" href="{{ printf "img/favicon/%s.png" $.Site.Params.ThemeColor | absURL }}">
{{ end }}
```

If we have `if/else` we don't need `with`. Lets go context free here ;-) (line 23 in sugestion is also very important)